deployment/embedded-spring-rest/pom.xml

```diff
@@ -10,9 +10,10 @@
 
   <properties>
     <version.camunda>7.16.0</version.camunda>
-    <version.spring>5.2.8.RELEASE</version.spring>
+    <version.spring>5.2.14.RELEASE</version.spring>
     <version.resteasy-jaxrs>3.14.0.Final</version.resteasy-jaxrs>
-    <version.h2>1.3.171</version.h2>
+    <version.h2>1.4.190</version.h2>
+    <version.slf4j>1.7.26</version.slf4j>
```

Comment:
Should we go for the latest versions here?
H2: 1.4.200
SLF4J: 1.7.32

Reply:
👍